pkg/notify/elasticsearch.go

```diff
@@ -91,7 +98,7 @@ func NewElasticSearch(c config.ElasticSearch) (Notifier, error) {
 	}
 	return &ElasticSearch{
 		ELSClient: elsClient,
-		Index:     c.Index.Name,
+		Index:     c.Index.Name + "-" + time.Now().Format(indexSuffixFormat),
```

Comment:
We create ElasticSearch object when BotKube starts - https://github.com/infracloudio/botkube/blob/develop/cmd/botkube/main.go#L59. If we create the index here, it will not be updated if BotKube stays running forever. We should re-calculated index (since it includes timestamp) each time when `SendEvent` is called

Reply:
refactored